src/app/components/core/asset-readings/readings-graph/readings-graph.component.html

```diff
@@ -93,9 +94,7 @@
                     [data]="assetReadingValues" [options]="assetChartOptions" #assetChart></app-chart>
                 </div>
                 <div *ngIf="selectedTab === 2 && arrayTypeReadingsList?.length > 0" data-content="2">
-                  <plotly-plot [data]="polyGraphData?.data" [layout]="polyGraphData?.layout"
-                    [config]="{displayModeBar: false}">
-                  </plotly-plot>
+                  <div id="graph" #Graph></div>
```

Comment:
Please name it   `<div id="3d_graph" #3dGraph></div>` or `<div id="graph_3d" #Graph3D></div>` so that in DOM and normally, it is clearly distinguishable!